AngularJS/scss/application.scss

```diff
@@ -0,0 +1,15 @@
+@import 'equalizer';
+@import 'colors';
+@import 'variables';
+@import 'navbar';
+@import 'dashboard';
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,400i,700&display=swap');
```

Comment:
The way you should organize the imports is:

- Equalizer
- Fonts and external imports
- Variables
- Commons